clang/lib/CodeGen/CGOpenMPRuntime.cpp

```diff
@@ -5928,8 +5931,10 @@ void CGOpenMPRuntime::emitTargetOutlinedFunctionHelper(
         return CGF.GenerateOpenMPCapturedStmtFunction(CS, D.getBeginLoc());
       };
 
-  OMPBuilder.emitTargetRegionFunction(EntryInfo, GenerateOutlinedFunction,
-                                      IsOffloadEntry, OutlinedFn, OutlinedFnID);
+  llvm::Error Err = OMPBuilder.emitTargetRegionFunction(
```

Comment:
_nit#2_: There is now some inconsistency in the use of `auto` type deduction. Here it isn't used but used previously in seemingly the same circumstances.